src/pages/ProfilePage/ProfilePage.js

```diff
@@ -1,12 +1,113 @@
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Menu from '../../components/Menu/Menu';
+
+const useStyles = makeStyles(theme => ({
+  paper: {
+    padding: theme.spacing(2),
+    color: theme.palette.text.secondary,
+  },
+  submitButton: {
+    marginTop: theme.spacing(2),
+  },
+}));
+
+export default function ProfilePage() {
+  const classes = useStyles();
+  const [userName, setUserName] = useState('Test Name Surname');
+  const [oldPassword, setOldPassword] = useState({ value: null, error: false, helperText: null });
+  const [newPassword, setNewPassword] = useState({ value: null, error: false, helperText: null });
+  const [repeatPassword, setRepeatPassword] = useState({ value: null, error: false, helperText: null });
+  const [daysLimit, setDaysLimit] = useState({ value: null, error: false, helperText: null });
+
+  useEffect(() => {
+    //TODO
+    // get user name and limit
+  }, []);
 
-function ProfilePage() {
   return (
     <div>
-    profile
+      <Menu>
+          <Grid container spacing={3}>
+            <Grid item xs={12}>
+              <Paper className={classes.paper}>{userName}</Paper>
+            </Grid>
+            <Grid item xs={6}>
+              <Paper className={classes.paper}>
+              Change password
+              <TextField
+                label="Current password"
+                fullWidth
+                required
+                autoFocus
+                type="password"
+                inputRef={input => oldPassword.value = input}
+                error={oldPassword.error}
+                helperText={oldPassword.helperText}
+              />
+              <TextField
+                label="New password"
+                fullWidth
+                required
+                autoFocus
+                type="password"
+                inputRef={input => newPassword.value = input}
+                error={newPassword.error}
+                helperText={newPassword.helperText}
+              />
+              <TextField
+                label="Repeat new password"
+                fullWidth
+                required
+                autoFocus
+                type="password"
+                inputRef={input => repeatPassword.value = input}
+                error={repeatPassword.error}
+                helperText={repeatPassword.helperText}
+              />
+              <Button
+                type="submit"
+                variant="contained"
+                color="primary"
+                raised
+                className={classes.submitButton}
+              >
+                Change
+              </Button>
+              </Paper>
+            </Grid>
+            <Grid item xs={6}>
+              <Paper className={classes.paper}>
+                Your monthly learning days limit
```

Comment:
If I understand correctly, this setting will only be accessible for the managers and should state: "Global monthly learning days limit for you and the people you manage". Would you agree with me?

Reply:
I think we should have not text field for user which would state 'Your monthly learning days limit'.
And for managers I think we should have separate field/table where manager could manage others limits. If you agree I can change this text field to something else, which would show the limit.

Reply:
Do agree with your proposed solution. Sorry for the late response.